app/src/ui/check-runs/ci-check-run-actions-job-step-item.tsx

```diff
@@ -44,7 +44,7 @@ export class CICheckRunActionsJobStepListItem extends React.PureComponent<ICIChe
     const { step } = this.props
     return (
       <li
-        className="ci-check-run-job-step list-item"
```

Comment:
You will notice I removed the `list-item` classes. I did this since they are functionally buttons not a list anymore and competing between existing list styles and button styles was hard.